src/styles/main.css

```diff
@@ -0,0 +1,197 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,500;1,400&display=swap');
+@import 'toggle.css';
+
+*,
+::after,
+::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+main {
+  margin-bottom: 2rem;
+}
+
+h1 {
+  text-transform: capitalize;
+  font-size: 2rem;
+}
+
+header {
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem 1rem;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.desktop-menu {
+  display: none;
+}
+
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #272a31;
+  opacity: 0.9;
+  backdrop-filter: blur(7px);
+  transition: all 1s ease;
+  display: none;
+}
```

Comment:
Great job with this mobile menu its working well. I think it would be better to make the close button on the Menu have a lighter color, so that it's more visible.
<img width="244" alt="mobileMenuCap" src="https://github.com/marvin-nyalik/Movie-Capstone/assets/81169099/c544afd2-ee5e-40e6-917a-eb1ca3ecd1e5">
As is seen, the close button is not clearly visible.